packages/oracle/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 1.25.1
```

Since we're updating the Kibana version, it's preferable to upgrade the minor version rather than the patch.
```suggestion
- version: 1.26.0
```

---

Agree!

---

Are these changes backward incompatible changes?

---

Yes, they are. The Old DSN would still continue to work. Please refer the comment down below and the details captured in the linked issue.